tests/py/dynamo/models/test_models_export.py

```diff
@@ -225,3 +225,50 @@ def calibrate_loop(model):
             )
             outputs_trt = trt_model(input_tensor)
             assert torch.allclose(output_pyt, outputs_trt, rtol=1e-3, atol=1e-2)
+
+
+@unittest.skipIf(
+    torch.cuda.get_device_properties(torch.cuda.current_device()).major < 9,
+    "int8 compilation in Torch-TRT is not supported on cards older than Hopper",
```

Comment:
INT8 should be supported on older cards as well. Is this a modelopt limitation ?